geode-core/src/main/java/org/apache/geode/management/internal/cli/result/ResultBuilder.java

```diff
@@ -56,17 +52,17 @@
    * @param message Message to be shown to the user
    * @return Result for connection error
```

Comment:
All the javadocs in here should now say the return type is CommandResult, not Result